### DISCUSSION TITLE
Model TAB, Dark Mode, Model names

### DISCUSSION THREAD

**Discussion by AdrianoAz** [civitai/civitai#225] [Ideas]
Hi, thank you opening this channel,

Would you mind in that menu horizontal tabs, put please, among that others like, sexy, photorealistic, anime, etc, etc, etc, the tab: MODELS, where the user can find easily only checkpoints? it's very annoying the Civitai browsing experience and becoming worst each time somebody put another waifu embedding or lora, I need browse too much not interesting stuff to find a model. 

This is only a sugestion, but for to say the real you would improve that menu is almost useless.

more one sugestion: the dark mode, I think not only me but other people vision must be compromissed, passing hours seeing blank screens

now an advice: Could be smart don't put name of real people in the models or embeddings or other technologies like Madonna, Rihanna, etc, etc, It's good at least put some warnings to avoid law problems.

Success!

**Comment by manuelurenah** [comment]
Hey @AdrianoAz! Thank you for the suggestions. You can find more advanced filters in the home page by clicking the filter icon just above the feed

![image](https://user-images.githubusercontent.com/12631159/218505078-f8547b94-f47e-4e28-b154-00ca18732a09.png)

With these filters, you can customize your feed by toggling between model types, SD versions, adult/safe content, etc.

Regarding the dark mode, you can click your avatar icon in the upper right corner of the page and toggle between light and dark modes

![image](https://user-images.githubusercontent.com/12631159/218505530-3c671422-2eea-45fd-94a9-cba407073720.png)

Hope all of this helps!

**Reply by ihsan62** [reply]
I  found it , thank you for your support
